### PR TITLE
Add test for empty CBOR data

### LINKED ISSUE #26: add test to check empty CBOR data from the profile decoder
if this is expected behaviour, we should have a test that covers this condition

_Originally posted by @thomas-fossati in https://github.com/veraison/eat/pull/22#discussion_r586257270_

### PR DESCRIPTION
**Pull request by yogeshbdeshpande**
Fixes #26

### CODE REVIEW

File: profile_test.go [hunk 1]
```diff
@@ -128,8 +128,28 @@ func TestProfile_MarshalCBOR_URL_OK(t *testing.T) {
 	assert.Equal(t, expected, actual)
 }
 
-// TestProfile_MarshalCBOR_URL_NOK tests the invalid case of CBOR marshaling of profile been set as an URI
-func TestProfile_MarshalCBOR_URL_NOK(t *testing.T) {
+// TestProfile_UnmarshalCBOR_URL_OK tests the valid case of CBOR Unmarshaling of profile claim set as URL
+func TestProfile_UnmarshalCBOR_URL_OK(t *testing.T) {
+	inputURL := "http://example.com"
+	profile, err := NewProfile(inputURL)
+	assert.Nil(t, err)
```
Comment (thomas-fossati) [inline profile_test.go:135]: this is technically a precondition, which should be a test requirement rather than a test assertion:
```suggestion
	require.assert.Nil(t, err)
```

**Review by thomas-fossati — COMMENTED** [review]
LGTM, I've left one comment

File: profile_test.go (hunk 1 above)
Comment (yogeshbdeshpande) [reply profile_test.go:135]: Can consider this enhancement as part of the Test Enhancement Issue #24

**Review by thomas-fossati — APPROVED** [review]
